exercises/difference-of-squares/difference_of_squares.py

```diff
@@ -1,10 +1,10 @@
-def square_of_sum():
+def square_of_sum(value):
```

Comment:
This is a best proposal I've seen (in comparison with #581 #585).
But still it's too general and does not reflect the nature of the input. Could you change it to something more meaningful?